### PR TITLE
Update to match latest DWS/NNF

### PR DESCRIPTION
**Pull request by roehrich-hpe**

### CODE REVIEW

File: nnf-container-example.yaml [hunk 1]
```diff
@@ -17,9 +17,10 @@ data:
           spec:
             containers:
               - name: nnf-container-example
-                image: ghcr.io/nearnodeflash/nnf-container-example:latest
+                image: ghcr.io/nearnodeflash/nnf-container-example:0.0.0.14-496e
```
Comment (bdevcich) [inline nnf-container-example.yaml]: Do we want latest here?

File: nnf-container-example.yaml [hunk 2]
```diff
@@ -33,15 +34,10 @@ data:
...
             containers:
               - name: nnf-container-example
-                image: ghcr.io/nearnodeflash/nnf-container-example:latest
-                # Note: nnf-sos now includes the proper capabilities by
-                # default. Leaving this here for now in case an older verion of
-                # nnf-sos is used.
-                # securityContext:
-                  # capabilities:
-                    # add: ["NET_BIND_SERVICE", "SYS_CHROOT", "AUDIT_WRITE", "SETUID", "SETGID"]
+                image: ghcr.io/nearnodeflash/nnf-container-example:0.0.0.14-496e
```
Comment (bdevcich) [inline nnf-container-example.yaml]: same

**Comment by bdevcich** [comment]
Thanks for updating this!

**Review by bdevcich — APPROVED** [review]

File: nnf-container-example.yaml (hunk 1 above)
Comment (roehrich-hpe) [reply nnf-container-example.yaml]: 'latest' is gone. I'm switching this repo to use docker/metadata-action.

Comment (roehrich-hpe) [reply nnf-container-example.yaml]: But we probably want 'master' here.

**Pull request merged**